WordPress/Classes/ViewRelated/Post/Preview/PreviewDeviceSelectionViewController.swift

```diff
@@ -37,7 +37,18 @@ class PreviewDeviceSelectionViewController: UIViewController {
         }
 
         var viewportScript: String {
-            return String(format: "let parent = document.querySelector('meta[name=viewport]'); parent.setAttribute('content', 'width=%1$d, initial-scale=0');", NSInteger(width))
+            let js = """
```

Comment:
I doubt this improves anything related to flickering, but it was an opportunity to clean things up:

- `initial-scale=0` may have been clamped to `0.1` [if I understand the spec correctly](https://www.w3.org/TR/css-device-adapt-1/#min-scale-max-scale).
- [some themes](https://bennettdemo.wordpress.com) have two viewport meta elements, so we wipe those now before overriding as the previous code would only select one

Reply:
Great idea! Also, I love the comments you've added! :+1: